validator-auction-static/js/chart.js

```diff
@@ -256,7 +246,16 @@ function getAuctionData() {
 
 $(window).on('load',
     function () {
-        getAuctionData()
+        getAuctionData(800)
```

Comment:
And here we can then write `fetchAuctionDataAndRender({animationDuration: 800})`